### DISCUSSION TITLE
MacOS?

### DISCUSSION THREAD

**Discussion by lilteddyp** [chuot/rdio-scanner#264] [Q&A] [answered]
Any plan to make a MacOS capable version?

**Comment by chuot** [comment, marked as answer]
There is already a macOS version available for Apple and Intel sillicons, but I agree the naming convention may be confusing to some.

Use the darwin version that suits your architecture from the [releases](https://github.com/chuot/rdio-scanner/releases) tab.

I will, however, change the naming convention for the next release so that macOS versions are named as such, as opposed to darwin as it is currently.

**Reply by lilteddyp** [reply]
Thank you for the reply and apologies for the ignorance. I’m more of a hardware guy than a software/programming guy lol. I will give it a shot and see if I can’t figure it all out. Thanks again.
